whylog/config/rule.py

```diff
@@ -57,6 +64,140 @@ def get_causes_parsers(self):
     def get_effect_name(self):
         return self._effect.name
 
+    def get_search_ranges(self, effect_clues):
+        group, group_type = self._effect.get_primary_key_group()
```

Comment:
`group` is a number, so maybe a better name for this variable is `group_number`? it has confused me.